cypress/support/commands/navigation.js

```diff
@@ -149,3 +149,29 @@ Cypress.Commands.add('selectRowByColumnAndValue', (columnName, expectedValue, mo
         });
     });
 });
+/**
+ * Select all rows on the current page
+ *
+ */
+Cypress.Commands.add('selectAllRowsOnCurrentPage', () => {
+  cy.get('body').type('{alt}a');
+});
+/**
+ * Select all rows from all pages
+ * This function only works on a list window, and not on a single view window
+ * @param windowId - the window for which the rows will be counted
+ */
+Cypress.Commands.add('countAllRows', windowId => {
+  cy.visitWindow(windowId);
+  cy.get('.pagination-row .pagination-part .hidden-sm-down').then(totalString => {
```

Comment:
I'm not really sure what's going on over here. We're counting empty spaces ? But hey, `if it's stupid but works, it's not stupid`. So we can leave it for now.

Reply:
It's because the value for total number of rows it's displayed like this in the DOM: 
![image](https://user-images.githubusercontent.com/50820854/64175273-a1042b80-ce63-11e9-8fce-e1001237ec55.png)